src/components/PhotoAdd/PhotoAdd.js

```diff
@@ -17,36 +20,40 @@ const PhotoAdd = ({updateProfilePicture, addGalleryPhoto, currentProfilePic, typ
         setPhotoFile(e.target.files[0])
     }
 
+    const updatePreview = () => {
+        if (type === "profile" && previewHeader.includes("Your")) {
+            setPhotoFilePath(currentProfilePic)
+            setPreviewHeader("Current Profile Picture")
+        }
+    }
+
     return (
-    <section className='photo-add'>
-        <div className="photo-small-square photo-corner1"></div>
-        {type=== "profile" && <div className="photo-title"><h2>Edit Profile Picture</h2></div>}
-        {type=== "gallery" && <div className="photo-title"><h2>Add a Photo to Gallery</h2></div>}
-        <div className="photo-small-square photo-corner2"></div>
-        <div className="photo-rectangle side1"></div>
-        <div className="square1">
-            {type==="profile" &&
-            <>
-                <p>{previewHeader}</p>
-                <img className="user-image" src={photoFilePath} alt="User Profile Picture"/>
-            </>
-            }
-        </div>
-        <div className="square2"></div>
-        <div className="photo-rectangle side2"></div>
-        <div className="photo-rectangle"></div>
-        <div className="square3"></div>
-        <article className="photo-form">
-            <p>Upload Your Photo Here</p>
-            <input className="photo-upload" type='file' accept="image/png, image/jpeg" onChange={(e) => {handleChange(e)}}></input>
-            <Link to={`/${type==="gallery" ? "gallery" : ""}`}><button className="photo-edit-button" onClick={(e) => handleSubmit(e)}>Submit</button></Link>
-        </article>
-        <div className="photo-rectangle"></div>
-        <div className="photo-small-square photo-corner3"></div>
-        <div className="photo-rectangle bottom-rectangle1"></div>
-        <div className="photo-rectangle bottom-rectangle2"></div>
-        <div className="photo-small-square photo-corner4"></div>
-    </section>
+        <section className='photo-add'>
+            <div className="photo-small-square photo-corner1"></div>
+            {type === "profile" && <div className="photo-title"><h2>Edit Profile Picture</h2></div>}
+            {type === "gallery" && <div className="photo-title"><h2>Add a Photo to Gallery</h2></div>}
```

Comment:
Nice job making this dynamic! Good idea to preview the image for the user before they add it to gallery.